README.gcp.md

```diff
@@ -0,0 +1,127 @@
+# DDS - Build, test, and deploy using the Google Cloup Platform (GCP)
+
+For project info please see README.md
+
+# Connect to the cloud shell
+
+Using your favorite web browser, open a cloud shell terminal at <https://ssh.cloud.google.com/cloudshell>.
+You'll need a free Google account. These days, most people have one already.
+
+# Build and install the C++ library
+
+This repository contains the hackathon's fork of Bo Haglund's double-dummy solver code.
+
+```
+gcloud config set project online-bridge-hackathon-2020
+mkdir ~/bridge-hackathon/
+cd ~/bridge-hackathon/
+git clone https://github.com/suokko/dds.git
+cd dds/
+mkdir .build
+cd .build/
+cmake ..
```

Comment:
What are the pros and cons? I hope we've now addressed the bugs in the C++ library, but if more turn up then whoever is debugging that can enable symbols.

Reply:
Debug symbols are quite cheap to add for potentially making thinks easier when C++ side decides to crash or have issues. Often people feel debug symbols are expensive because they are combined with reduced optimization flags for compiler. But at least I have learned to debug fully optimized applications if I just have debug symbols. I could even debug binaries without symbols but time spent doing that is like at least 10 times more than with symbols available.

Cost is practically only a bigger binary size with debug symbol sections not loaded into memory when normally running application. Sections are mapped to the memory but pages are loaded from disk lazily only if a debug tool actually uses those pages.

On other hand you can get find debug symbols useful if there is crash or deadlock. Then you can just analyze resulting core file or attach debugger to deadlocked process. Then there is even a case where core and binary with debug symbols could be sent to someone else for analyze.

Then you can also find it helpful to have symbols if you use profiling tools like `perf record`.

Reply:
Done.